src/Acuminator/Acuminator.Runner.NetFramework/Input/CommandLineOptions.cs

```diff
@@ -0,0 +1,381 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Acuminator.Runner.Constants;
+
+using CommandLine;
+
+using Microsoft.CodeAnalysis;
+
+using Serilog.Events;
+
+namespace Acuminator.Runner.Input
+{
+	internal class CommandLineOptions
+	{
+		/// <summary>
+		/// The code source that will be analysed by Acuminator.
+		/// </summary>
+		/// <remarks>
+		/// Currently, the supported code sources are C# projects and C# solutions.
+		/// </remarks>
+		[Value(index: 0, MetaName = CommandLineArgNames.CodeSource, Required = true,
+			   HelpText = """
+			A path to the "code source" which will be validated. The term "code source" is a generalization for components/services that can provide source code to the tool.
```

Comment:
@EAndrosova please review all strings assigned to `HelpText` properties in this file. They describe tool's setting when `--help` command is specified. Here is how it looks:
![image](https://github.com/user-attachments/assets/97c25c02-202b-48bb-a4eb-4032fba3e306)